### PR TITLE
docs: Added process for How to rename a repo

### LINKED ISSUE #194: [DOCS] Adding 'Renaming a Repository' in the How To section
### Description

Hi! @Pradumnasaraf, I wanted to add a piece of useful info on how to rename a repository in the how-to section of your website, as it can be a life-saver for a lot of beginners. 

Kindly assign me this issue if you feel like this will work. Thanks in advance!

### PR DESCRIPTION
**Pull request by nexi9**
## 🛠️ Fixes Issue

 Closes #194 

## 👨‍💻 Changes proposed

- Added rename-repo.md, which explains how to rename a repository.
- Added the image link in README.md of How-to section.

## ✔️ Check List 

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

<a href="https://gitpod.io/#https://github.com/Pradumnasaraf/open-source-with-pradumna/pull/210"></a>

### CODE REVIEW

File: pages/How-to/guide/rename-repo.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+[![p4.png](https://i.postimg.cc/q7CXRwZd/p4.png)](https://postimg.cc/5HbFKBVP)
+
+
+#### Hurray, you just renamed your repository 🎉  
+
+
+
+------------
+
```
Comment (Pradumnasaraf) [inline pages/How-to/guide/rename-repo.md:41]:
```suggestion
```

File: pages/How-to/guide/rename-repo.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+Under the Repository Name heading, type the new name of your repository.
+
+[![p3.png](https://i.postimg.cc/R0Q4qHdR/p3.png)](https://postimg.cc/qtR95g9h)
+
+### Step 4 :
+Click Rename. 
+
+[![p4.png](https://i.postimg.cc/q7CXRwZd/p4.png)](https://postimg.cc/5HbFKBVP)
+
+
```
Comment (Pradumnasaraf) [inline pages/How-to/guide/rename-repo.md:35]:
```suggestion
```

File: pages/How-to/guide/rename-repo.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+Whenever you rename a repository, keep in mind certain elements will also be associated with the new name, such as:
+- Issues
+- Wikis
+- Stars 
+- Followers
+
+
+------------
+
```
Comment (Pradumnasaraf) [inline pages/How-to/guide/rename-repo.md:11]:
```suggestion
```

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Thank you for the PR. Please remove the extra line and space.

Also can add a line at the bottom, that after you rename the repo, your old repo link will redirect to the new repo link.

**Comment by nexi9** [comment]
@Pradumnasaraf kindly check if the changes are satisfactory

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

**Pull request merged**